cli/src/config-schema.json

```diff
@@ -439,6 +439,11 @@
                     "type": "string",
                     "description": "Path to the git executable",
                     "default": "git"
+                },
+                "ignore-lfs-files": {
+                    "type": "boolean",
+                    "description": "Whether jj ignores files handled by git lfs or not",
+                    "default": false
```

Comment:
I would advocate for defaulting this to `true`. In a repo that has `git lfs` tracked files, it will be an exceedingly rare exception that the user wants `jj` to track these files.

Reply:
Thinking about what possible use cases there could be for this behavior some more, I would go as far as to say that setting this to false is never useful practice: Tracking lfs files will immediately remove their lfs property, so this is effectively the same as running `git lfs untrack .`. This makes a value of false here just a roundabout way of removing `git lfs` from a repo, an action that the user will much more likely want to perform explicitly as lfs untrack && lfs uninstall.